Makefile

```diff
@@ -55,7 +55,11 @@ test:
 	export LOG_LEVEL=DEBUG && \
 	export RUST_BACKTRACE=1 && \
 	cargo test --features "${ENABLE_FEATURES}" ${NO_RUN} -- --nocapture && \
-	cargo test --features "${ENABLE_FEATURES}" --bench benches ${NO_RUN} -- --nocapture 
+	cargo test --features "${ENABLE_FEATURES}" --bench benches ${NO_RUN} -- --nocapture && \
+	if [[ "`uname`" == "Linux" ]]; then \
```

Comment:
using leak check may be better? 
Do we have any test in tikv-server binary now?